tests/functional/modules/test_zos_fetch_func.py

```diff
@@ -696,13 +705,13 @@ def test_fetch_partitioned_data_set_replace_on_local_machine(ansible_zos_module)
 
 def test_fetch_uss_file_insufficient_write_permission_fails(ansible_zos_module):
     hosts = ansible_zos_module
-    dest_path = "/tmp/profile"
+    dest_path = get_unique_uss_file_name()
     with open(dest_path, "w",encoding="utf-8") as dest_file:
         dest_file.close()
     os.chmod(dest_path, stat.S_IREAD)
     params = {
         "src":"/etc/profile",
-        "dest":"/tmp/",
+        "dest": dest_path,
```

Comment:
When creating temporary files that are going to be used in the local node is better that we use `tempfile.NamedTemporaryFile` with `delete=True` and `delete_on_close=False`, that will delete the file once the context is closed, make sure to remove the file since is not automatically removed.